R/misc.R

```diff
@@ -457,3 +457,13 @@ simple_terms <- function(x, ...) {
   res
 }
 
+check_name <- function(x, y, name) {
```

Comment:
Maybe pass in the step object and use the first class to write the warning to avoid typos